### DISCUSSION TITLE
Project search

### DISCUSSION THREAD

**Discussion by sdrshn** [community/community#21924] [New to GitHub]
<p>Am new to github and just uploaded a first repository.  When i go to the GitHub search box and search directly for the phrase Theme Creation CMS that I have in  the top header portion of the readme file,   seven repositories  show up but the one I put up is not one of them.  Did I miss something??  How does one address keywords?</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/sdrshn">@sdrshn</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Let me know if you have any other questions or if I can help with anything else.</p>
<p>Cheers!</p>

**Comment by sdrshn** [comment, marked as answer]
<p> Ok saw the topics feature and that solves it.</p>
